webapp/src/main/java/org/apache/atlas/notification/EntityNotificationListenerV2.java

```diff
@@ -307,6 +307,16 @@ private AtlasEntityHeaderWithRelations toNotificationHeader(AtlasEntity entity)
                 }
             }
 
+            // fill the internal properties like __glossary, __meanings etc. (ES Isolation)
+            RequestContext context = RequestContext.get();
+            Map<String, Object> allInternalAttributesMap = context.getAllInternalAttributesMap().get(entity.getGuid());
+            if (MapUtils.isNotEmpty(allInternalAttributesMap)) {
+                for (String key : allInternalAttributesMap.keySet()) {
+                    Object value = allInternalAttributesMap.get(key);
+                    ret.setAttribute(key, value);
```

Comment:
this will put internal attributes in attributes like `attributes.__meanings`. Nothing wrong in this just wanted to highlight